.github/workflows/eslint.yml

```diff
@@ -0,0 +1,21 @@
+name: javascript
+on: push
+jobs:
+    build:
+      name: eslint
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v1
+        - uses: actions/setup-node@v1
+          with:
+            node-version: '10.x'
```

Comment:
Why not more precision?

Reply:
We could, how much precision do we want?

Reply:
Ideally this should match whatever we have in `.nvmrc`, right? Otherwise I don't really care much as long as we get the latest and it is in line with everything else :)

Reply:
Yeah, more node versions to change :(